### PR TITLE
Bugfixes - This branch is used for out of scope small bugfixes, refactors etc.

### PR DESCRIPTION
**Pull request by quansenB**
# Description

This PR formats all files with our prettier config file, fixes small bugs and refactors small parts of the code

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

File: controllers/user/resetPassword.js [hunk 1]
```diff
@@ -49,22 +47,14 @@ const resetPassword = async (req, res, next) => {
...
 
-    if (!user) {
-      return next(
-        createError({
-          message: 'Please login, to reset password',
-          status: UNAUTHORIZED
-        })
-      );
-    }
-
```
Comment (Rexben001) [inline controllers/user/resetPassword.js]: Why did you remove line 59 - 66?

Comment (quansenB) [reply controllers/user/resetPassword.js]: For one, because a missing token is already detected by the verifyToken middle ware. 
Secondly there is already the same check done in line 24-31

File: middlewares/isAuthorized.js [hunk 2]
```diff
@@ -9,10 +9,12 @@ const { createError, FORBIDDEN } = require('../helpers/error');
  */
 const isAuthorized = (req, res, next) => {
   if (!req.user.isAdmin) {
-    return next(createError({
-      message: 'Forbidden, you don\'t have access!',
-      status: FORBIDDEN
-    }));
+    return next(
+      createError({
+        message: "Forbidden, you don't have access!",
```
Comment (daquinons) [inline middlewares/isAuthorized.js]: Single quotes pls! 👍

File: models/changeRequest.js [hunk 3]
```diff
@@ -1,4 +1,3 @@
-/* eslint-disable camelcase */
```
Comment (daquinons) [inline models/changeRequest.js]: In general I would say that all the removal of eslint lines are not positive changes. They are there because if not, the linter complains. This for example
image

But there were other changes like this one. For the eslint line to disappear, in this case it would also need to be changed to use camel case, but I don't recommend it because it changes the format of the response... So my suggestion would be to leave the eslint lines as they were

Comment (quansenB) [reply models/changeRequest.js]: I just noticed that my eslint was disabled. I'll reinclude the exception comments

Comment (quansenB) [reply models/changeRequest.js]: Implemented changes
